circus-train-housekeeping/src/test/java/com/hotels/bdp/circustrain/housekeeping/LegacyReplicaPathRepositoryTest.java

```diff
@@ -0,0 +1,118 @@
+package com.hotels.bdp.circustrain.housekeeping; /**
```

Comment:
Similar here, I think we should have a configurable base DB implementation that can be used more or less "out of the box" in the housekeeping module that has all these tests etc. Otherwise this has to be repeated in each downstream project.